libsolidity/formal/SolverInterface.h

```diff
@@ -44,18 +44,18 @@ enum class CheckResult
 
 enum class Sort
 {
-	Int, Bool
+	Int, Bool, IntIntFun
```

Comment:
Does `IntIntFun` means it is a function with a single `Int` input and single `Int` output? Would a comment here help?